internal/scorecard/alpha/scorecard.go

```diff
@@ -175,10 +206,10 @@ func (r PodTestRunner) RunTest(ctx context.Context, test Test) (*v1alpha3.TestSt
 // RunTest executes a single test
 func (r FakeTestRunner) RunTest(ctx context.Context, test Test) (result *v1alpha3.TestStatus, err error) {
 	select {
+	case <-time.After(r.Sleep):
```

Comment:
This may also be a problem in `FakeTestRunner`'s `Initialize` and `Cleanup` methods.

Reply:
What's the problem?

Reply:
Actually neither of them need to consider `r.Sleep`, disregard.